README.md

```diff
@@ -1,49 +1,177 @@
-| Easy install | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
-| Easy pack upgrade | ✅ | ✅ | ✅ | ❌ | ✅ | ❌ |
-| Easy pack upgrade for newer MC | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ |
-| Add mods with few clicks | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ |
-| Supports the modpack and mods | ✅ | ✅ | ●¹ | ●¹ | ●¹ | ●¹ |
-| Can be used offline | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| Windows, macOS, Linux support | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| Unique feature | 1-click install from site | Import from other launchers | Steam Deck support | Nostalgic UI | Always up to date | Familiarity |
-
-¹ Supports the modpack by increasing its download count, but not with monetary benefits.
-
-<details>
-  <summary>Alternative downloads</summary>
-
-_These downloads may not fully support the modpack or the mods :(_
-* [MultiMC (auto-update)](https://wiki.download.fo/readme/multimc-auto-update): [1.16.5](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.16.5.auto-update.zip) | [1.17.1](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.2.3/Fabulously.Optimized.MC.1.17.1.auto-update.zip) | [1.18.2](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v3.12.2/Fabulously.Optimized.MC.1.18.2.auto-update.zip) | [1.19.4](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v4.10.0/Fabulously.Optimized.MC.1.19.4.auto-update.zip) | [1.20.2](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.6.5/Fabulously.Optimized.MC.1.20.2.auto-update.zip) | [1.20.4](https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/v5.8.0-beta.11/Fabulously.Optimized.MC.1.20.4.auto-update.zip)
-* [GitHub releases](https://github.com/Fabulously-Optimized/fabulously-optimized/releases)
-* Chat Reporting Helper resource pack: [CurseForge](https://www.curseforge.com/minecraft/texture-packs/chat-reporting-helper) | 
-[Modrinth](https://modrinth.com/resourcepack/chat-reporting-helper) |
-[Planet Minecraft](https://www.planetminecraft.com/texture-pack/chat-reporting-helper) (bundled in FO)
-* Fast Better Grass resource pack: [CurseForge](https://www.curseforge.com/minecraft/texture-packs/fast-better-grass) | [Modrinth](https://modrinth.com/resourcepack/fast-better-grass) | [Planet Minecraft](https://www.planetminecraft.com/texture-pack/fast-better-grass) (bundled in FO)
-</details>
+<img src="https://download.fo/icon.png" alt="FO logo" align="right" width="10%" height="10%"/>
+
+# Fabulously Optimized
+
+[![download.fo][img-website]][url-website]
+[![Wiki][img-wiki]][url-wiki]
+[![❤️ Support Us!][img-thanks]][url-thanks]
+
+[![Discord][img-discord]][url-discord]
+[![GitHub][img-github]][url-github]
+[![Mastodon][img-mastodon]][url-mastodon]
+[![Crowdin][img-translate]][url-translate]
```

Comment:
- Yes, it's a bit more complicated, and it might be approximate, because it uses the invite link
- Agreed. Should it say "Help us translate"? And... should it be in the `README.md`, since it's already in the Thanks page?

Reply:
* LGTM.
* Yes - people should have an easy link to it in the readme, because otherwise they may assume that making a pull request is the expected way to do it.